src/engine/Server.cpp

```diff
@@ -841,6 +842,55 @@ Awaitable<void> Server::processQuery(
   co_return;
 }
 
+// ____________________________________________________________________________
+Awaitable<void> Server::processUpdate(
+    const ad_utility::url_parser::ParamValueMap& params, const string& update,
+    ad_utility::Timer& requestTimer,
+    const ad_utility::httpUtils::HttpRequest auto& request, auto&& send,
+    TimeLimit timeLimit) {
```

Comment:
Do we already limit UPDATEs to users who have the valid access token? You probably know better, if so, that was in one of the old PRs.

Reply:
We don't at the moment. But with the previous PRs it should be as easy as adding `requireValidAccessToken("Update")`.